index.html

```diff
@@ -89,7 +89,8 @@ <h1 class="header intro-header">I’m Jordan Glad to see you!</h1>
             </div>
         </section>
 
-        <section id="section-2" class="padding">
+
+        <section id="section-2" class="padding">                  
```

Comment:
- [ ] Great job implementing the pop-up, it looks amazing and the design is very similar to the original one 👏  According to the requirements, this whole section should be created dynamically, using the information stored in the JavaScrip object, in your case `popupContent`, you can checkout more here:

![image](https://user-images.githubusercontent.com/66967274/146474860-7b552948-9708-4fee-b208-f3ba18cc66fb.png)
> This will help you keep your portfolio updated, because whenever you need to update a project, all you have to do is update the data from the projects array!

Reply:
Thank you so much